### ISSUE TITLE
Making HttpTransactions externally viewable

### ISSUE THREAD

**Issue by tkdgusl94** [ChuckerTeam/chucker#1001]
## :warning: Is your feature request related to a problem? Please describe

Hi there! 
In my project, I want to create another logging tool using HttpTransaction.
However, since HttpTransaction is an internal class, it is currently not externally accessible.
I hope to be able to access the HttpTransaction data.

## :bulb: Describe the solution you'd like

I want to be able to view the HttpTransaction data externally.
## :bar_chart: Describe alternatives you've considered

Because HttpTransaction is an internal class, it cannot be accessed externally. I don't think it's correct to change it to a public class.
I'm currently using HttpTransaction and HttpTransactionTuple from the library's HttpTransactionRepository.
I create a class called A that maps to HttpTransaction, and then open only class A as public.

## :raising_hand: Do you want to develop this feature yourself?

- [x] Yes
- [ ] No

**Comment by cortinico** [comment]
Thanks for reporting this. I'll close this as it's effectively a duplicate of https://github.com/ChuckerTeam/chucker/issues/259
Once we land the rework of the internals, you'll be able to access the `HttpTransaction` class.
